modules/cstrike/cstrike/CstrikeNatives.cpp

```diff
@@ -1944,17 +1944,66 @@ static cell AMX_NATIVE_CALL cs_get_weapon_info(AMX* amx, cell* params)
 	return 0;
 }
 
-// native cs_get_user_weapon_entity(playerIndex);
+// native cs_get_user_weapon_entity(playerIndex, weaponIndex = CSW_NONE);
 static cell AMX_NATIVE_CALL cs_get_user_weapon_entity(AMX *amx, cell *params)
 {
-	GET_OFFSET("CBasePlayer", m_pActiveItem);
+	GET_OFFSET("CBasePlayer"    , m_pActiveItem);
```

Comment:
I know it doesn't matter, but I would move the `GET_OFFSET` where it's actually used and after any checks.